TimeTable/TimeTable/Program.cs

```diff
@@ -0,0 +1,152 @@
+            Database databaseObject = new Database();
+            
+            // View all saved events
+            string queryAllEvents = "SELECT * FROM myCalendar ORDER BY eventDate ASC";
+            SQLiteCommand selectAllCommand = new SQLiteCommand(queryAllEvents, databaseObject.myConnection);
+            databaseObject.OpenConnection();
+            SQLiteDataReader resultAll = selectAllCommand.ExecuteReader();
+            if (resultAll.HasRows)
+            {
+                while (resultAll.Read())
+                {
+                    Console.WriteLine("Event Data: {0} - Event Name: {1}", resultAll["eventDate"], resultAll["eventName"]);
+                }
+            }
+            ShowMenu();
+        }
+
+        // Display selected date
+        static void SelectOneDay()
+        {
+            // Ask the user to enter any date
+            string enteredDay;
+            Console.WriteLine("Enter the date in the format YYYY-MM-DD: ");
+            enteredDay = Console.ReadLine();
+
+            ShowEventsByDay(enteredDay);
+            ShowMenu();
+        }
+
+        static void ShowEventsByDay(string enteredDay)
+        {
+            // Search and display the selected date
+            Database databaseObject = new Database();
+
+            string query = $"SELECT * FROM myCalendar WHERE eventDate='{enteredDay}'";
+
+            SQLiteCommand selectOneDayCommand = new SQLiteCommand(query, databaseObject.myConnection);
+            databaseObject.OpenConnection();
+            SQLiteDataReader resultFromOneDay = selectOneDayCommand.ExecuteReader();
+            if (resultFromOneDay.HasRows)
+            {
+                while (resultFromOneDay.Read())
+                {
+                    Console.WriteLine("Event Data: {0} - Event Name: {1}", resultFromOneDay["eventDate"], resultFromOneDay["eventName"]);
+                }
+            }
+            else
+            {
+                Console.WriteLine();
+                Console.WriteLine("You have no events for this day");
+            }
+            
+         }
+
+        static void AddEvents()
+        {
+            // Add event to The calendar
+            Database databaseObject = new Database();
+
+            string query = "INSERT INTO myCalendar (`eventDate`, `eventName`) VALUES (@eventDate, @eventName)";
+            SQLiteCommand myCommand = new SQLiteCommand(query, databaseObject.myConnection);
+
+            databaseObject.OpenConnection();
+
+            // Ask the user to enter a date 
+            string eventDate;
+            Console.WriteLine("Enter the day in the format YYYY-MM-DD: ");
+            eventDate = Console.ReadLine();
+            myCommand.Parameters.AddWithValue("@eventDate", eventDate);
+
+            // Ask user to enter a Event Name
+            string eventName;
+            Console.WriteLine("Enter a name for the event: ");
+            eventName = Console.ReadLine();
```

Comment:
Maybe create model (class) which instances will represent Event with fields Name and Date. You can add validation of date there.

Reply:
I will think about it in the future when I improve the cod to be more object oriented, thank you.